src/søknad/steg/4-barnasbosted/SkalBarnBoHosDeg.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import KomponentGruppe from '../../../components/gruppe/KomponentGruppe';
+import { useIntl } from 'react-intl';
+import { IBarn } from '../../../models/person';
+import { skalBarnBoHosDeg } from './ForeldreConfig';
+import FeltGruppe from '../../../components/gruppe/FeltGruppe';
+import { AlertStripeAdvarsel, AlertStripeInfo } from 'nav-frontend-alertstriper';
+import MultiSvarSpørsmål from '../../../components/spørsmål/MultiSvarSpørsmål';
+import { Normaltekst } from 'nav-frontend-typografi';
+import LocaleTekst from '../../../language/LocaleTekst';
+
+interface Props {
+    barn: IBarn;
+    forelder: any;
+    settForelder: Function;
+}
+
+const SkalBarnBoHosDeg: React.FC<Props> = ({ barn, forelder, settForelder }) => {
+    const intl = useIntl();
+
+  return (
+    <>
+        {!barn.harSammeAdresse ? <KomponentGruppe>
+    <AlertStripeAdvarsel className={'fjernBakgrunn'}>
+        <LocaleTekst tekst={'barnasbosted.alert.måBoHosDeg'} />
+    </AlertStripeAdvarsel>
+    <MultiSvarSpørsmål
+            key={skalBarnBoHosDeg.søknadid}
+            spørsmål={skalBarnBoHosDeg}
+            valgtSvar={forelder.skalBarnBoHosDeg}
+            settSpørsmålOgSvar={(_, svar) => settForelder({...forelder, [skalBarnBoHosDeg.søknadid]: svar})}
+        />
+        </KomponentGruppe>              
+        : null}
+    {forelder.skalBarnBoHosDeg === intl.formatMessage({ id: 'barnasbosted.spm.jaMenSamarbeiderIkke' }) ? 
+    <FeltGruppe>
+    <AlertStripeInfo className={'fjernBakgrunn'}>
+        <LocaleTekst tekst={'barnasbosted.alert.hvisFaktiskBor'} />
+    </AlertStripeInfo>
+    <FeltGruppe>
+    <Normaltekst className="innskutt">Familievernkontoret kan også hjelpe deg</Normaltekst>
+    </FeltGruppe>
+    <Normaltekst className="innskutt">Når det kommer til denne søknaden kan du dokumentere at Lise bor hos deg ved å sende inn for eksempel:</Normaltekst>
+    <ul className="dokumentere-bosted">
+        <li><Normaltekst>redegjørelse for årsaken til manglende adresseendring for barnet</Normaltekst></li>
+        <li><Normaltekst>kopi av flyttemelding/tips til Folkeregisteret</Normaltekst></li>
+        <li><Normaltekst>bekreftelse fra for eksempel barnehage/skole, barnevern eller helsestasjon</Normaltekst></li>
+    </ul>
```

Comment:
intl-tekster bør kanskje brukes her? :)  
Et forslag er å bruke html tags i tekster.ts for å formattere teksten til å se ut som du vil. 
Da må du bruke <FormattedHTMLMessage id='tekstid' /> for at den skal klare å lese html-tagsa. 

Dette er bare en midlertidig løsning tho. 
Blir sikkert bedre når vi får inn Sanity igjen for de lengre tekstene. 😪 
Kan vurdere om det er verdt tiden 🤷‍♂

Reply:
Samme for `HvordanPraktiseresSamværet.tsx`

Reply:
Ja, tenkte at vi bare kunne ha disse lengre tekstene her siden de sannsynligvis skal inn i Sanity senere uansett, så kan vente med disse.